prql-compiler/tests/integration/main.rs

```diff
@@ -1,20 +1,38 @@
 // Re-enable on windows when duckdb supports it
 // https://github.com/wangfenjin/duckdb-rs/issues/62
 #![cfg(not(any(target_family = "windows", target_family = "wasm")))]
+// TODO enable it for all OS
+#![cfg(target_os = "linux")]
```

Comment:
```suggestion
```

Together with the CI change, I think we can remove these now (tested locally, it works great!)